x-pack/plugins/infra/server/plugin.ts

```diff
@@ -167,6 +167,9 @@ export class InfraServerPlugin {
     // Telemetry
     UsageCollector.registerUsageCollector(plugins.usageCollection);
 
+    const logEntriesService = new LogEntriesService();
+    logEntriesService.setup(core, { ...plugins, sources });
```

Comment:
Trying to match the [platform conventions](https://github.com/elastic/kibana/blob/7c8b9a1f0059dfe7522c73933eb982b06a3fbe7d/src/core/CONVENTIONS.md) regarding the substructure of plugins.